### DISCUSSION TITLE
Enabling 'tee-sys' from within fixture/test?

### DISCUSSION THREAD

**Discussion by Tbruno25** [pytest-dev/pytest#9570] [Q&A]
I've got a certain test which relies on `--capture=tee-sys` being enabled. 

Wondering if there is any way of enabling this functionality from a module level regardless of whether the option was set or not at the cli level?

**Comment by nicoddemus** [comment]
Hi @Tbruno25,

This is not currently possible I'm afraid, `--capture` is set in the CaptureManager right at the start of the session, and can't be changed afterwards.

I can think of these solutions to at least remedy the situation:
1. Check that during `pytest_configure` to at least error out with an appropriate message, rather than having tests failing. 
2. Skip those tests which require `tee-sys` if `--capture` is not the required mode.
